### ISSUE TITLE
Bug: Visibility icon used incorrectly

### ISSUE THREAD

**Issue by AkhilaSunesh** [Counselllor/Counsellor-Web#721]
**Describe the bug**
When we click on the eye button the password should be seen. T
The current button does the exact opposite
![image](https://github.com/Counselllor/Counsellor-Web/assets/140897461/57186345-be0e-4dd1-84fe-0bde6b91079b)

Kindly assign this issue to me.

**Comment by thestarsahil** [comment]
@AkhilaSunesh Would you like to Raise More PR and we are accepting PR of those contributors who repeadelty contributing to this project !

**Comment by AkhilaSunesh** [comment]
Yeah...once I will be assigned with this issue. I will raise more prs
